src/data/blog/v-vvv-e-eee.md

```diff
@@ -0,0 +1,53 @@
+---
+title: 'Value and effort and prioritisation.'
```

Comment:
I don't think we need a full stop in the title. I'd say use title case but we haven't consistently used that before..

Reply:
Thank you!